pipeline/01_parser.py

```
@@ -235,12 +400,26 @@ def parse_docx(file_path):
+                    txt = cell.text.strip()
+                    if txt and txt not in seen_cells:
+                        cells.append(txt)
+                        seen_cells.add(txt)
+                if not cells:
+                    continue
+                if row_idx == 0:
+                    header_cells = cells
+                    result.append("[표헤더] " + " | ".join(cells))
+                else:
+                    if header_cells and len(header_cells) == len(cells):
+                        parts = [f"{h}: {v}" for h, v in zip(header_cells, cells)]
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)